Mesh/d4est_mesh.c

```diff
@@ -1346,14 +1346,14 @@ d4est_mesh_compute_l2_norm_sqr
 
 
 double
-d4est_mesh_compute_linf
+d4est_mesh_compute_linf // TODO: remove
```

Comment:
Can we just remove this? The L_infty norm (without the skip function) is simple enough to be computed in `d4est_norms`, it doesn't even need any context.

Reply:
Yep, sounds good